CertificateTests/CertificateTest.cs

```diff
@@ -0,0 +1,169 @@
+            CertificatePage.FillCertificateForm(FilePath);
+            Thread.Sleep(4000);
+            CertificatePage.CertAssertion(expectedText);
+        }
+
+        [Test, Order(6), Description("This test validates behavior of Certificate feature for  no input in year field")]
+        public void TestCase006()
+        {
+            string expectedText ="Please enter Certification Name, Certification From and Certification Year";
+            const string FilePath = @"C:\Users\Shuch\Desktop\NunitCompetition\Utilities\CertificateTestCases\TestCase006.json";
+            CertificatePage.FillCertificateForm(FilePath);
+            Thread.Sleep(4000);
+            CertificatePage.CertAssertion(expectedText);
+        }
+
+        [Test, Order(7), Description("This test validates behavior  of Certificate feature for  no input in certificate field")]
+        public void TestCase007()
+        {
+            string expectedText = "Please enter Certification Name, Certification From and Certification Year";
+            const string FilePath = @"C:\Users\Shuch\Desktop\NunitCompetition\Utilities\CertificateTestCases\TestCase007.json";
+            CertificatePage.FillCertificateForm(FilePath);
+            Thread.Sleep(4000);
+            CertificatePage.CertAssertion(expectedText);
+        }
+        [Test, Order(8), Description("This test validates behavior  of Certificate feature for  no input in From field.")]
+        public void TestCase008()
+        {
+            string expectedText = "Please enter Certification Name, Certification From and Certification Year";
+
+            const string FilePath = @"C:\Users\Shuch\Desktop\NunitCompetition\Utilities\CertificateTestCases\TestCase008.json";
+            CertificatePage.FillCertificateForm(FilePath);
+            Thread.Sleep(4000);
+            CertificatePage.CertAssertion(expectedText);
+        }
+
+        [Test, Order(9), Description("This test validates behavior  of Certificate feature for duplicate certigficate entries.")]
+        [TestCase(@"C:\Users\Shuch\Desktop\NunitCompetition\Utilities\CertificateTestCases\TestCase009.json")]
+        public void TestCase009(string jsonFilePath)
+        {
+            // Expected validation message for duplicate entries
+            string expectedText = "This information is already exist.";
+
+            // First attempt to add the entry
+            CertificatePage.FillCertificateForm(jsonFilePath);
+            Thread.Sleep(2000);
+
+            // Second attempt to add the same entry
+            CertificatePage.FillCertificateForm(jsonFilePath);
+            Thread.Sleep(1000);
+            CertificatePage.CertAssertion(expectedText); // Assert duplicate entry message
+        }
+
+
+
+        [TearDown]
+
+        public void Cleanup()
+        {
+            // Perform cleanup after each test
+            Console.WriteLine("Test cleanup executed.");
+            CertificatePage.DeleteAllElement(driver);
```

Comment:
i would say for this do not delete all the elements but only the test data that test is adding e.g test 1 added english and french as language test 2 added hindi then when you reset data you already know what language you are adding for each test
there for on reset of the two test you should delete english and french for test 1 and hindi for test 2